components/invitation_modal/invitation_modal.jsx

```diff
@@ -88,6 +93,15 @@ export default class InvitationModal extends React.Component {
         }
     }
 
+    confirmBack = () => {
+        this.setState({step: STEPS_INITIAL, hasChanges: false});
+        this.setState({confirmBack: false});
```

Comment:
Can this be combined into a single invocation of `setState`?

Reply:
Sure

Reply:
Done